packages/clerk-js/src/ui/components/ApiKeys/CreateApiKeyForm.tsx

```
@@ -1,62 +1,131 @@
-import React, { useState } from 'react';
+import React, { useLayoutEffect, useMemo, useRef, useState } from 'react';
```

ESLint

[error] 'useLayoutEffect' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)